const generalErrorMessage = 'エラーが発生しました。';
const generalExceptionMessage = 'エラーが発生しました。';
const networkNotConnected = 'ネットワーク接続がありません。';
